Program.fs

```diff
@@ -83,17 +80,9 @@ module MainModule =
         return body
     }
 
-    let midnight () = 
-        let now = DateTime.Now
-        let midnight = DateTime (now.Year, now.Month, now.Day, 0, 0, 0)
+    let midnight () = DateTime.Now.Date
```

Comment:
Good to know! Dates were always something that I had trouble with, I'm never happy working with them. 😄